TransformationSystem/Client/TransformationCLI.py

```diff
@@ -14,6 +14,7 @@
 
 from DIRAC.TransformationSystem.Client.Transformation           import Transformation
 from DIRAC.TransformationSystem.Client.TransformationClient     import TransformationClient
+from DIRAC.Resources.Catalog.FileCatalogClient                  import FileCatalogClient
```

Comment:
I would suggest running pylint on these files, it'd spot these type of things immediately.